### PR TITLE
Add funds page

### PR DESCRIPTION
**Pull request by eluyema**
There is no functionality yet, mostly only layout

### CODE REVIEW

File: frontend/src/common/enums/routes.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ export enum Routes {
   HOME = '/',
   LOGIN = '/login',
   SIGNUP = '/signup',
+  ADDFUNDS = '/addfunds',
```
Comment (yevhenii-moroziuk) [inline frontend/src/common/enums/routes.ts]: Let's use /add-funds

File: frontend/src/components/Routing/index.tsx [hunk 2]
```diff
@@ -27,10 +29,13 @@ const Routing = () => {
 
   return (
     <Switch>
+      <Route path={Routes.HOME} exact component={Main} />
+      <Route path={Routes.ADDFUNDS} exact component={FundsPage} />
+      <Route path={Routes.LOGIN} exact component={LoginPage} />
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Routing/index.tsx]: Just add Funds page. It will be private page.

File: frontend/src/components/Routing/index.tsx [hunk 3]
```diff
@@ -27,10 +29,13 @@ const Routing = () => {
...
       <PublicRoute restricted={false} path={Routes.HOME} exact component={Main} />
       <PublicRoute restricted={false} path={Routes.LOGIN} exact component={LoginPage} />
     </Switch>
   );
 };
-
+//      <Route path={Routes.HOME} exact component={Main} />
```
Comment (yevhenii-moroziuk) [inline frontend/src/components/Routing/index.tsx]: comment

File: frontend/src/scenes/Wallet/CustomFund/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Form } from 'react-bootstrap';
+import coinImg from '../../../assets/HypeCoin.png';
+import classes from './styles.module.scss';
+
+const CustomFund: FC = () => {
+  const { t, i18n } = useTranslation();
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Wallet/CustomFund/index.tsx]: Use here useLocalization hook (You need to merge develop branch). Also, you need to update LocaleKeys type, just run command: `generate:keys` in frontend directory.

File: frontend/src/scenes/Wallet/CustomFund/index.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  const onInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (validationPrice(event.target.value)) setPrice(event.target.value);
+  };
+  return (
+    <div className={classes['wrp-fund-body']}>
+      <Form className={classes['fund-body']} onSubmit={onSubmit}>
+        <div className={classes['input-wrp']}>
+          <h2> {t('Custom fund')}</h2>
+          <Form.Control
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Wallet/CustomFund/index.tsx:22]: Let's use our Input component

File: frontend/src/scenes/Wallet/CustomFund/index.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            onInput={onInput}
+            value={price}
+            type="number"
+            step="0.01"
+            min="0.05"
+            max="100000"
+          />
+        </div>
+
+        <button type="submit"> {t('Add funds')}</button>
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Wallet/CustomFund/index.tsx]: use our button

File: frontend/src/scenes/Wallet/Fund/styles.module.scss [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+        border-radius: 6px;
+        margin-right: 3px;
+      }
+    }
+    &:hover {
+      background-color: rgba(0, 0, 0, 0.4);
+    }
+  }
+  &:hover {
+    background: #12c2e9; /* fallback for old browsers */
```
Comment (yevhenii-moroziuk) [inline frontend/src/scenes/Wallet/Fund/styles.module.scss:52]: Did you copy the styles from somewhere?

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/Wallet/Fund/styles.module.scss (hunk 7 above)
Comment (eluyema) [reply frontend/src/scenes/Wallet/Fund/styles.module.scss:52]: I use generator uigradients.com for gradient background

File: frontend/src/components/Routing/index.tsx (hunk 2 above)
Comment (eluyema) [reply frontend/src/components/Routing/index.tsx]: What it's mean?

Comment (yevhenii-moroziuk) [reply frontend/src/components/Routing/index.tsx]: It seems, that you added 3 new routes (Home, AddFunds, Login) and imported Switch the second time. I guess it was a merge conflict. Just add only 1 page, and wrap it in private route:
 `<PrivateRoute path={Routes.ADDFUNDS} exact component={FundsPage} />`

File: frontend/src/scenes/Wallet/CustomFund/index.tsx (hunk 5 above)
Comment (eluyema) [reply frontend/src/scenes/Wallet/CustomFund/index.tsx:22]: Our input component not have type for number

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**